### ISSUE TITLE
Teams v2, klicking on Teams channel make app freeze

### ISSUE THREAD

**Issue by joakim-tjernlund** [IsmaelMartinez/teams-for-linux#1206]
**Describe the bug**
In Teams V2 mode I click on Teams channels icon and then teams-for-linux just freeze, I can only Quit 
and restart teams. In version 1.4.21 I also get an popup window that says: "You have no access" or similar
but I cannot close that window.

**To Reproduce**
1. Click on Teams icon
2. Teams freezes

**Expected behavior**
I should see the various Teams channels I have and post in them

**Desktop (please complete the following information):**
 - OS: Uptodate Gentoo
 - Installation package teams-for-linux ebuild
 - Version 1.4.21 or 1.4.28

**Debug**

The provide in this section the output from both the terminal and the browser debug console.

```bash
teams-for-linux --webDebug --appLogLevels=error,info,warn,debug
[DEBUG] configPath:,/home/jocke/.config/teams-for-linux
[WARN] No config file found, using default values
[DEBUG] configFile:,[object Object]
[DEBUG] Fetching custom background configuration from 'http://localhost/config.json'
[DEBUG] Custom background service url is 'http://localhost'
[DEBUG] processArgs:,/opt/teams-for-linux/teams-for-linux,--lite-mode,--optimize-for-size,--webDebug,--appLogLevels=error,info,warn,debug
[DEBUG] Testing network using net.request() for https://teams.microsoft.com/
[DEBUG] Spellchecker is disabled!
[3757816:0418/151027.946461:ERROR:browser_main_loop.cc(280)] Gtk: gtk_widget_add_accelerator: assertion 'GTK_IS_ACCEL_GROUP (accel_group)' failed
[3757816:0418/151027.946588:ERROR:browser_main_loop.cc(280)] Gtk: gtk_widget_add_accelerator: assertion 'GTK_IS_ACCEL_GROUP (accel_group)' failed
[3757816:0418/151027.946631:ERROR:browser_main_loop.cc(280)] Gtk: gtk_widget_add_accelerator: assertion 'GTK_IS_ACCEL_GROUP (accel_group)' failed
[3757816:0418/151027.946669:ERROR:browser_main_loop.cc(280)] Gtk: gtk_widget_add_accelerator: assertion 'GTK_IS_ACCEL_GROUP (accel_group)' failed
[3757816:0418/151027.947421:ERROR:browser_main_loop.cc(280)] Gtk: gtk_widget_add_accelerator: assertion 'GTK_IS_ACCEL_GROUP (accel_group)' failed
[ERROR] Failed to fetch custom background remote configuration. net::ERR_CONNECTION_REFUSED
[DEBUG] Network test successful with method https
[3757816:0418/151029.025812:ERROR:CONSOLE(1)] "Request Autofill.enable failed. {"code":-32601,"message":"'Autofill.enable' wasn't found"}", source: devtools://devtools/bundled/core/protocol_client/protocol_client.js (1)
[DEBUG] GetSystemIdleState => IdleTimeout: 300s, IdleTimeoutPollInterval: 10s, ActiveCheckPollInterval: 2s, IdleTime: 0s, IdleState: 'active'
[DEBUG] did-frame-finish-load,[object Object],true
[DEBUG] did-finish-load
[DEBUG] GetSystemIdleState => IdleTimeout: 300s, IdleTimeoutPollInterval: 10s, ActiveCheckPollInterval: 2s, IdleTime: 0s, IdleState: 'active'
[DEBUG] did-frame-finish-load,[object Object],true
[DEBUG] did-finish-load
[DEBUG] GetSystemIdleState => IdleTimeout: 300s, IdleTimeoutPollInterval: 10s, ActiveCheckPollInterval: 2s, IdleTime: 0s, IdleState: 'active'
[DEBUG] did-frame-finish-load,[object Object],true
[DEBUG] did-finish-load
[3757816:0418/151038.057038:ERROR:CONSOLE(1)] "Request Autofill.enable failed. {"code":-32601,"message":"'Autofill.enable' wasn't found"}", source: devtools://devtools/bundled/core/protocol_client/protocol_client.js (1)
[DEBUG] did-frame-finish-load,[object Object],false
[DEBUG] GetSystemIdleState => IdleTimeout: 300s, IdleTimeoutPollInterval: 10s, ActiveCheckPollInterval: 2s, IdleTime: 6s, IdleState: 'active'

```
[teams.microsoft.com-1713445974445.log](https://github.com/IsmaelMartinez/teams-for-linux/files/15025381/teams.microsoft.com-1713445974445.log)

**Comment by IsmaelMartinez** [comment]
Hi @joakim-tjernlund , a wee question on your description. What do you mean with ebuild? Are you doing your own build? Can you try the deb package? Thanks!

**Comment by IsmaelMartinez** [comment]
Hi, sorry but I don't have experience with ebuild and we already provide a lot of different packaging options. I will leave it open to see if anyone else can help, but otherwise I will recommend to use one of the many packages we provide. Hope helps!

**Comment by joakim-tjernlund** [comment]
This is nothing odd I think, I download the provided  teams-for-linux-1.4.28.tar.gz and unpack it.
The error is odd though, I asked around at work and I am the only one seeing this problem.

**Comment by IsmaelMartinez** [comment]
maybe the chromium version you are running or one of the libraries? Or a proxy is playing around with you if you are working from home and your co-workers aren't

**Comment by joakim-tjernlund** [comment]
> maybe the chromium version you are running or one of the libraries? Or a proxy is playing around with you if you are working from home and your co-workers aren't

No, it the same problem from home and work. Different computers too.
But today I tried using Google Chrome instead and swa the same problem :(
So I guess this is not a teams-for-linux problem but a MS Teams v2 one.

**Comment by joakim-tjernlund** [comment]
One difference in Google Chome: I get "You don't have access to this channel"  popup window but
the page times out and I get a new pop up:
"Page is Unresponsive" and I can choose "Exit Page" and regain control again.
But as soon as I choose Teams channels again I get the same error.

I deleted lots of old channels of not interest anymore but I still get the same error

**Comment by IsmaelMartinez** [comment]
I think this then is a MS issue. I think you might have a bit more luck in https://feedbackportal.microsoft.com/feedback/forum/ad198462-1c1c-ec11-b6e7-0022481f8472 . Maybe clean the cache of the app in case is somehow trying to open the latest channel you had open, and that was the one you just deleted and is not handling that situation gracefully. 

See https://github.com/IsmaelMartinez/teams-for-linux/blob/develop/KNOWN_ISSUES.md#no-history on how to delete the cache (or history). Don't worry, you don't really delete the data as that is in the MS servers, only the downloaded cache.

**Comment by MTijn** [comment]
Same here, noticed before that I had the same issue in Firefox, but that seems to have been resolved now.

Sometimes it does show the pop-up with "You don't have access to this channel" and sometimes I do not, however it does seem to take a lot of CPU load.

- Cleared the cache by deleting ~/.config/teams-for-linux but that doesn't solve it.
- Tried different versions, from 1.4.17 to 1.4.29, all have the same issue
- Chromium seems to have this issue too (Version 123.0.6312.122)
- Firefox 125.0.2 (64-bit) does not

But can't find any log entries that would point towards the issue at all.

**Comment by joakim-tjernlund** [comment]
> Same here, noticed before that I had the same issue in Firefox, but that seems to have been resolved now.

Can one use Firefox now in Teams v2 without loosing features? Did MS v2 remove PWA?

**Comment by joakim-tjernlund** [comment]
Updated to google-chrome-beta-125.0.6422.4 and got one error msg I could dismiss and now 
the Teams channel work :)
Logs form chrome:
```
google-chrome-beta
[375987:375987:0423/143123.259636:ERROR:viz_main_impl.cc(166)] Exiting GPU process due to errors during initialization
[376127:376127:0423/143124.017466:ERROR:viz_main_impl.cc(166)] Exiting GPU process due to errors during initialization
[375933:375933:0423/143125.846662:ERROR:CONSOLE(0)] "Refused to execute script from 'https://www.google.com/chrome/static/js/main.v2.min.js' because its MIME type ('text/html') is not executable, and strict MIME type checking is enabled.", source: https://www.google.com//chrome/404 (0)
Created TensorFlow Lite XNNPACK delegate for CPU.
Warning: DeviceDescriptor.deviceLostCallback and DeviceDescriptor.deviceLostUserdata are deprecated. Use DeviceDescriptor.deviceLostCallbackInfo instead.
Warning: DeviceDescriptor.deviceLostCallback and DeviceDescriptor.deviceLostUserdata are deprecated. Use DeviceDescriptor.deviceLostCallbackInfo instead.
```

**Comment by MTijn** [comment]
> > Same here, noticed before that I had the same issue in Firefox, but that seems to have been resolved now.
> 
> Can one use Firefox now in Teams v2 without loosing features? Did MS v2 remove PWA?

It seems so, in the past I wasn't able to call in Firefox, but in V2 it seems that I can do that.

Not sure what which wouldn't work, selecting backgrounds seems to work too, together mode is visible for me too.

**Comment by MTijn** [comment]
Interesting enough it seems that Microsoft solved it, saw the pop-up for the error message, but was able to click it away and continue without any issues.

**Comment by joakim-tjernlund** [comment]
> Interesting enough it seems that Microsoft solved it, saw the pop-up for the error message, but was able to click it away and continue without any issues.

Yes, same for me even in teams-for-linux. I guess we can close this now

**Comment by jijojosephk** [comment]
Closing as this seems like a temporary issue.
